### PR TITLE
Added Landing Screen, Notifications Screen, and updated all Onboarding Pages

### PR DESCRIPTION
**Pull request by ckunchur**
# *Name of the PR*

## :recycle: Current situation & Problem
*Describe the current situation (if possible, with an exemplary (or real) code snippet and/or where this is used)*

I successfully implemented a conditionally rendered landing screen and updated all of the onboarding page content to walk through the app features, consent, and health data sharing permissions. I also implemented a notifications screen to give a preview of what upload status notifications will look like. 

ISSUE: 
After creating a fake ECG data point in the Simulator's Health App, I am unable to see the FHIR encoded JSON in the "Mock Upload" tab in the PAWS app, even after rebuilding, deleting/relaunching the app, etc.

The potential sources of error include the current method of binding for checking if onboarding was completed and  a change was made 2 commits ago that permanently sets the completedOnboarding variable to be false. However, it is unclear why the onboarding process would be related to the mock upload tab's ability to display data. 

*Please link any open issue that is addressed with this PR*
Shriya's PR has successful viewing of the JSON file in the mock upload tab and should be used as a point of comparison for detecting differences.

## :bulb: Proposed solution
*Describe the solution and how this affects the project and internal structure*

## :gear: Release Notes 
*Add a summary of the feature and possible migration guides if this is a breaking change so this section can be added to the release notes.*
*Include code snippets that provide examples of the feature implemented if it appends or changes the public interface.*

## :heavy_plus_sign: Additional Information
*Provide some additional information if possible*

### Related PRs
*Reference the related PRs*
Shriya's PR has successful viewing of the JSON file in the mock upload tab and should be used as a point of comparison for detecting changes. 

### Testing
*Are there tests included? If yes, which situations are tested, and which corner cases are missing?*

### Reviewer Nudging
*Where should the reviewer start? Where is a good entry point?*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md):
- [ +] I agree to follow the [Code of Conduct](https://github.com/CS342/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/CS342/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: PAWS/PAWS.swift [hunk 1]
```diff
@@ -10,22 +10,41 @@ import CardinalKit
...
+                } else {
+                    LandingScreen(pressedStart: $pressedStart)
                 }
-                .testingSetup()
+            }
+                    .sheet(isPresented: isSheetPresented) {
+                        OnboardingFlow()
+                    }
+                    .interactiveDismissDisabled(true)
+                    .testingSetup()
```
Comment (PSchmiedmayer) [inline PAWS/PAWS.swift]: Please indent any modifiers 4 spaces (or one tab) lower than the modified view (like the `.cardinalKit(appDelegate)` modifier):
```suggestion
                .sheet(isPresented: isSheetPresented) {
                    OnboardingFlow()
                }
                .interactiveDismissDisabled(true)
                .testingSetup()
```

File: PAWS/PAWS.swift [hunk 2]
```diff
@@ -10,22 +10,41 @@ import CardinalKit
...
+            }, set: { _ in }
+        )
+    }
     
     var body: some Scene {
         WindowGroup {
-            HomeView()
-                .sheet(isPresented: !$completedOnboardingFlow) {
-                    OnboardingFlow()
+//            if  completedAccountSetup {
```
Comment (PSchmiedmayer) [inline PAWS/PAWS.swift]: Please remove any code that is commented out to have a clean codebase:
```suggestion
```

File: PAWS/PAWS.swift [hunk 3]
```diff
@@ -10,22 +10,41 @@ import CardinalKit
...
 import PAWSSharedContext
 import SwiftUI
-
+import PAWSLandingScreen
 
 @main
 struct PAWS: App {
     @UIApplicationDelegateAdaptor(PAWSAppDelegate.self) var appDelegate
     @AppStorage(StorageKeys.onboardingFlowComplete) var completedOnboardingFlow = false
+//    @AppStorage(StorageKeys.accountCreated) var completedAccountSetup = false
```
Comment (PSchmiedmayer) [inline PAWS/PAWS.swift]: Please remove any code that is commented out to have a clean codebase:
```suggestion
```

File: PAWS/Home.swift [hunk 4]
```diff
@@ -9,27 +9,34 @@
...
     
     var body: some View {
         TabView(selection: $selectedTab) {
-            ScheduleView()
-                .tag(Tabs.schedule)
+            //            ScheduleView()
+            //                .tag(Tabs.schedule)
+            //                .tabItem {
+            //                    Label("SCHEDULE_TAB_TITLE", systemImage: "square.and.arrow.up")
+            //                }
```
Comment (PSchmiedmayer) [inline PAWS/Home.swift]: Please remove any code that is commented out to have a clean codebase:
```suggestion
            //            ScheduleView()
            //                .tag(Tabs.schedule)
            //                .tabItem {
            //                    Label("SCHEDULE_TAB_TITLE", systemImage: "square.and.arrow.up")
            //                }
```

File: PAWS/Home.swift [hunk 5]
```diff
@@ -9,27 +9,34 @@
...
 import PAWSSchedule
+import PAWSNotificationScreen
 import PAWSSharedContext
 import SwiftUI
 
 
 struct HomeView: View {
     enum Tabs: String {
-        case schedule
+        //case schedule
```
Comment (PSchmiedmayer) [inline PAWS/Home.swift]: Please remove any code that is commented out to have a clean codebase:
```suggestion
```

File: PAWS/PAWSAppDelegate.swift [hunk 6]
```diff
@@ -24,7 +24,20 @@ class PAWSAppDelegate: CardinalKitAppDelegate {
...
+                    )
+                    CollectSample(
+                        HKQuantityType(.heartRateVariabilitySDNN),
+                        deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+                    )
+                    CollectSample(
+                        HKQuantityType(.restingHeartRate),
+                        deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+                    )
+
```
Comment (PSchmiedmayer) [inline PAWS/PAWSAppDelegate.swift]: Please remove the empty line here:
```suggestion
```

File: PAWS/PAWSAppDelegate.swift [hunk 7]
```diff
@@ -24,7 +24,20 @@ class PAWSAppDelegate: CardinalKitAppDelegate {
             if HKHealthStore.isHealthDataAvailable() {
                 HealthKit {
                     CollectSample(
-                        HKQuantityType(.stepCount),
+                        HKQuantityType(.heartRate),
+                        deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
```
Comment (PSchmiedmayer) [inline PAWS/PAWSAppDelegate.swift:27]: I think it might even make sense to get all the HealthKit data in the background as we will most likely want to upload them even if the user did not open the app frequently:
```suggestion
                        deliverySetting: .background(.afterAuthorizationAndApplicationWillLaunch)
```

File: PAWSModules/Sources/PAWSContacts/Contacts.swift [hunk 8]
```diff
@@ -16,31 +16,64 @@ public struct Contacts: View {
...
+                address.city = "Palo Alto"
+                address.street = "725 Welch Rd"
                 return address
             }(),
             contactOptions: [
-                .call("+1 (650) 723-2300"),
-                .text("+1 (650) 723-2300"),
-                .email(addresses: ["contact@stanford.edu"]),
+                //.call("+1 (650) 723-2300"),
+                //.text("+1 (650) 723-2300"),
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSContacts/Contacts.swift]: Please remove any commented out lines:
```suggestion
```

File: PAWSModules/Sources/PAWSContacts/Contacts.swift [hunk 9]
```diff
@@ -16,31 +16,64 @@ public struct Contacts: View {
...
+                address.country = "USA"
+                address.state = "CA"
+                address.postalCode = "94304"
+                address.city = "Palo Alto"
+                address.street = "725 Welch Rd"
+                return address
+            }(),
+            contactOptions: [
+                //.call("+1 (650) 723-2300"),
+                //.text("+1 (650) 723-2300"),
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSContacts/Contacts.swift]: Please remove any commented out lines:
```suggestion
```

File: PAWSModules/Sources/PAWSContacts/Resources/en.lproj/Localizable.strings [hunk 10]
```diff
@@ -8,5 +8,5 @@
 
 // MARK: - Contacts
 "CONTACTS_NAVIGATION_TITLE" = "Contacts";
-"LELAND_STANFORD_BIO" = "Amasa Leland Stanford (March 9, 1824 – June 21, 1893) was an American industrialist and politician. [...] He and his wife Jane were also the founders of Stanford University, which they named after their late son.
-[https://en.wikipedia.org/wiki/Leland_Stanford]";
+"SCOTT_CERESNAK_BIO" = "Dr. Scott R. Ceresnak is a pediatric cardiologist in Walnut Creek, California and is affiliated with multiple hospitals in the area, including California Pacific Medical Center and Stanford Health Care-Stanford Hospital. He received his medical degree from Rutgers Robert Wood Johnson Medical School and has been in practice for more than 20 years.";
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSContacts/Resources/en.lproj/Localizable.strings:11]: Nice that you already added some bios here! 🚀

File: PAWSModules/Package.swift [hunk 11]
```diff
@@ -22,7 +22,11 @@ let package = Package(
         .library(name: "PAWSMockDataStorageProvider", targets: ["PAWSMockDataStorageProvider"]),
         .library(name: "PAWSOnboardingFlow", targets: ["PAWSOnboardingFlow"]),
         .library(name: "PAWSSchedule", targets: ["PAWSSchedule"]),
-        .library(name: "PAWSSharedContext", targets: ["PAWSSharedContext"])
+        .library(name: "PAWSSharedContext", targets: ["PAWSSharedContext"]),
+        .library(name: "PAWSLandingScreen", targets: ["PAWSLandingScreen"]),
+        .library(name: "PAWSNotificationScreen", targets: ["PAWSNotificationScreen"]),
+
+
```
Comment (PSchmiedmayer) [inline PAWSModules/Package.swift]: Please remove any unused/empty lines:
```suggestion
```

File: PAWSModules/Package.swift [hunk 12]
```diff
@@ -73,6 +77,27 @@ let package = Package(
         .target(
             name: "PAWSSharedContext",
             dependencies: []
+        ),
+        .target(
+            name: "PAWSLandingScreen",
+            dependencies: [],
+            resources: [
+                .process("Resources")
+            ]
```
Comment (PSchmiedmayer) [inline PAWSModules/Package.swift]: You can omit the dependencies list and the resources definition if a target does not use them or has no resources to resolve the warnings in your project:
```suggestion
            name: "PAWSLandingScreen"
```

File: PAWSModules/Package.swift [hunk 13]
```diff
@@ -73,6 +77,27 @@ let package = Package(
...
+        ),
+        .target(
+            name: "PAWSNotificationScreen",
+            dependencies: [],
+            resources: [
+                .process("Resources")
+            ]
```
Comment (PSchmiedmayer) [inline PAWSModules/Package.swift]: You can omit the dependencies list and the resources definition if a target does not use them or has no resources to resolve the warnings in your project:
```suggestion
            name: "PAWSNotificationScreen"
```

File: PAWSModules/Package.swift [hunk 14]
```diff
@@ -73,6 +77,27 @@ let package = Package(
...
+        ),
+        .target(
+            name: "PAWSRegisterScreen",
```
Comment (PSchmiedmayer) [inline PAWSModules/Package.swift]: Rename the folder that contains the sources to `PAWSRegisterScreen` to resolve the warning from the Swift Package Manger here.

File: PAWSModules/Package.swift [hunk 15]
```diff
@@ -73,6 +77,27 @@ let package = Package(
...
+            dependencies: [],
```
Comment (PSchmiedmayer) [inline PAWSModules/Package.swift]: You can remove empty dependency definitions if you don't use a dependency:
```suggestion
            name: "PAWSRegisterScreen"
```

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 16]
```diff
@@ -0,0 +1,69 @@
...
+import SwiftUI
+
+let backgroundGradient =  LinearGradient(
+    colors: [.red, .pink, .orange, .yellow],
+        startPoint: .leading,
+        endPoint: .trailing
+    )
+
+public struct LandingScreen: View {
+//    @AppStorage(StorageKeys.accountCreated) var completedAccountSetup = false
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]: Please remove any unused code:
```suggestion
```

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 17]
```diff
@@ -0,0 +1,69 @@
...
+    @Binding private var launchStatus: Bool
+    public var body: some View {
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]: I do generally recommend adding two empty lines between different groups of elements in types, e.g. stored properties (`launchStatus`) or computed properties `body`:
```suggestion
    @Binding private var launchStatus: Bool
    
    
    public var body: some View {
```

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 18]
```diff
@@ -0,0 +1,69 @@
...
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+let backgroundGradient =  LinearGradient(
+    colors: [.red, .pink, .orange, .yellow],
+        startPoint: .leading,
+        endPoint: .trailing
+    )
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]: Please move any properties that are related to a view within the type definition of the type and e.g. make them private if they are not used outside of the view.

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 19]
```diff
@@ -0,0 +1,69 @@
...
+                        .frame(width: 200, height: 200)
+                        .foregroundColor(.red)
+                    Text("PAWS")
+                        .font(.custom("GillSans-Bold", fixedSize: 30))
+                    Text("The Pediatric Apple Watch Study")
+                        .font(.custom("GillSans", fixedSize: 15))
+                        .offset(y: 50)
+                    
+                   
+
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]: Pleaser remove multiple empty lines in view definitions:
```suggestion
```

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 20]
```diff
@@ -0,0 +1,69 @@
...
+                
+                Text("Tap to get started")
+                    .fontWeight(.bold)
+                    .padding()
+                    .foregroundColor(Color.red)
+                    .border(backgroundGradient, width: 3)
+                    .cornerRadius(5)
+                    .offset(y: -30)
+            }
+
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]: I recommend the following formatting for a better readability of buttons and SwiftUI views in general:
```suggestion
            Button(
                action: {
                    launchStatus = true
                },
                label: {
                    Text("Tap to get started")
                        .fontWeight(.bold)
                        .padding()
                        .foregroundColor(Color.red)
                        .border(backgroundGradient, width: 3)
                        .cornerRadius(5)
                        .offset(y: -30)
                }
            )
```

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 21]
```diff
@@ -0,0 +1,69 @@
...
+        }
+    }
+    public init(pressedStart: Binding<Bool>) {
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]: I do generally recommend adding two empty lines between different groups of elements in types, e.g. computed properties (body) and initializers:
```suggestion
    }
    
    
    public init(pressedStart: Binding<Bool>) {
```

File: PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift [hunk 22]
```diff
@@ -0,0 +1,69 @@
...
+        self._launchStatus = pressedStart
+    }
+}
+
+
+struct LandingScreen_Previews: PreviewProvider {
+    @State private static var pressedStart: Bool = false
+    
+    static var previews: some View {
+
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift]:
```suggestion
```

File: PAWSModules/Sources/PAWSNotificationScreen/Notification.swift [hunk 23]
```diff
@@ -0,0 +1,65 @@
...
+// This source file is part of the CS342 2023 PAWS Team Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+let backgroundGradient =  LinearGradient(
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSNotificationScreen/Notification.swift]: The same types of comments as found in the `PAWSModules/Sources/PAWSLandingScreen/LandingScreen.swift` file also apply to this file:
- I would suggest to move properties inside the views they are related to.
- Please remove any unused code
- I do generally recommend adding two empty lines between different groups of elements in types, e.g. computed properties (body) and initializers
- Try to format your code using indentions for better readability

File: PAWSModules/Sources/PAWSNotificationScreen/NotificationScreen.swift [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+                    .padding([.top, .bottom], 20)
+                Notification()
+                Notification()
+                Notification()
+                Notification()
+            }
+        }
+        
+    }
+    public init(){}
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSNotificationScreen/NotificationScreen.swift]:
```suggestion
    }
    
    
    public init(){}
```

File: PAWSModules/Sources/PAWSOnboardingFlow/Consent.swift [hunk 25]
```diff
@@ -23,20 +23,33 @@ struct Consent: View {
...
+                .foregroundColor(.red)
+                .offset(y: 10)
+            ConsentView(
+                header: {
+                    OnboardingTitleView(
+                        title: "CONSENT_TITLE".moduleLocalized,
+                        subtitle: "CONSENT_SUBTITLE".moduleLocalized
+                    )
+                },
+
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSOnboardingFlow/Consent.swift:40]:
```suggestion
```

File: PAWSModules/Sources/PAWSOnboardingFlow/Consent.swift [hunk 26]
```diff
@@ -23,20 +23,33 @@ struct Consent: View {
...
+                    consentDocument
+                },
+                action: {
+                    onboardingSteps.append(.healthKitPermissions)
+                }
+            )
+            .offset(y: -20)
+        }
+        
+
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSOnboardingFlow/Consent.swift]:
```suggestion
```

File: PAWSModules/Sources/PAWSOnboardingFlow/HealthKitPermissions.swift [hunk 27]
```diff
@@ -22,6 +22,12 @@ struct HealthKitPermissions: View {
         OnboardingView(
             contentView: {
                 VStack {
+                    Image(systemName: "pawprint.circle.fill")
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSOnboardingFlow/HealthKitPermissions.swift:25]: I like the customizations that you mad here! 🚀

File: PAWSModules/Sources/PAWSOnboardingFlow/InterestingModules.swift [hunk 28]
```diff
@@ -15,37 +15,44 @@ struct InterestingModules: View {
...
+                actionText: "INTERESTING_MODULES_BUTTON".moduleLocalized,
+                action: {
+#if targetEnvironment(simulator) && (arch(i386) || arch(x86_64))
+                    print("PKCanvas view-related views are currently skipped on Intel-based iOS simulators due to a metal bug on the simulator.")
+                    onboardingSteps.append(.healthKitPermissions)
+#else
+                    //                onboardingSteps.append(.register)
+                    onboardingSteps.append(.consent)
+                    
+#endif
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSOnboardingFlow/InterestingModules.swift]: Please indent your code on the same level and remove unused code:
```suggestion
                    #if targetEnvironment(simulator) && (arch(i386) || arch(x86_64))
                    print("PKCanvas view-related views are currently skipped on Intel-based iOS simulators due to a metal bug on the simulator.")
                    onboardingSteps.append(.healthKitPermissions)
                    #else
                    onboardingSteps.append(.consent)
                    #endif
```

File: PAWSModules/Sources/PAWSOnboardingFlow/OnboardingFlow.swift [hunk 29]
```diff
@@ -15,6 +15,7 @@ public struct OnboardingFlow: View {
     enum Step: String, Codable {
         case interestingModules
         case consent
+//        case register
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSOnboardingFlow/OnboardingFlow.swift]:
```suggestion
```

File: PAWSModules/Sources/PAWSOnboardingFlow/OnboardingFlow.swift [hunk 30]
```diff
@@ -23,19 +24,29 @@ public struct OnboardingFlow: View {
...
+                    .offset(y: 20)
+                Welcome(onboardingSteps: $onboardingSteps)
+                    .navigationDestination(for: Step.self) { onboardingStep in
+                        switch onboardingStep {
+                        case .interestingModules:
+                            InterestingModules(onboardingSteps: $onboardingSteps)
+                        case .consent:
+                            Consent(onboardingSteps: $onboardingSteps)
+                            //                        case .register:
+                            //                            Register(onboardingSteps: $onboardingSteps)
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PAWSOnboardingFlow/OnboardingFlow.swift]: Please remove any unused code when you merge into the main branch.
```suggestion
```

File: PAWSModules/Sources/PawsRegisterScreen/RegisterScreen.swift [hunk 31]
```diff
@@ -0,0 +1,77 @@
...
+// This source file is part of the CS342 2023 PAWS Team Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+struct RegisterScreen: View {
```
Comment (PSchmiedmayer) [inline PAWSModules/Sources/PawsRegisterScreen/RegisterScreen.swift]: We will provide you with a screen to register in the template application as soon as we have the firebase connect in place. You can then use that for the registration here 🚀

Comment (PSchmiedmayer) [inline PAWSModules/Sources/PawsRegisterScreen/RegisterScreen.swift]: Apple also provides included Sign-In-With apple Buttons that you will be able to use. We will make sure that this is setup with Firebase for you. You can remove the Sign in with Apple image for now.

**Review by PSchmiedmayer — APPROVED** [review]
Hi @ckunchur,

A great job with this PR! 
Amazing to see what you and the team have achieved so far.

I have added a lot of comments in the code to improve it and facilitate its readability in the future. Sorry for the number of comments; I tried to make them very easy fixes by providing review suggestions in almost all of them to incorporate them more easily.

I have identified a few problems with you HealthKit Module when debugging the code and I have created a PR in the main CardinalKit repo to address this: https://github.com/StanfordBDHG/CardinalKit/pull/46 ... once this is done you should be able to update to the new version and should get all the bug fixes including a fix that will reenable the collection of HealthKit data for you.

You can easily resolve a large number of SwiftLint comments using the `swiftlint --fix` command from your command line in the root of your project (where the `.swiftlint.yml` file is located).

Your next steps would be:
- Address the different comments
- Resolve the SwiftLint warnings (most of them can be resolved automatically using swiftlint --fix)
- Address the REUSE specification errors and adjust the UI tests to reflect the new onboarding flow
- Merge the PR, I have approved the PR, and you can merge them after all the issues above are resolved 🥳🚀

I am happy to help with any of these steps; let me know if and how I can assist you there!

**Comment by PSchmiedmayer** [comment]
@ckunchur I took a look at the failing tests and modified them a bit to reflect the new structure in the application. You can see the diff here: https://github.com/CS342/2023-PAWS/pull/4/commits/5091cdb3b636ca13cbd91828c9f08f3da0414cb0

Regarding the failing UI test for the Mock Upload: The UI test was entering step count information into the Health App using the `try exitAppAndOpenHealth(.steps)` call. As you have changed the app delegate to no longer collect steps the application was no longer listening to changes in step count in the Health App (HealthKit) and therefore did not trigger an upload in the Mock Upload part of the app.
I changed the UI test to enter an ECG (the app collects that type of data) and now the test passes again.

In addition I improved a few things that I saw here and there with the following comments:
- https://github.com/CS342/2023-PAWS/pull/4/commits/969fc4e68265b6b8ce6600612f781af15a3c3e52
- https://github.com/CS342/2023-PAWS/pull/4/commits/1034441532383ce315c69abc8d56bd054926ee05

The UI tests should now pass and you should be able to merge the branch in the main branch once the coverage report comes that will now include the reenabled UI tests.

**Comment by PSchmiedmayer** [comment]
@ckunchur Short update here: There was a problem related to git LFS (similar in origin to https://github.com/orgs/CS342/discussions/29, but not the same) that I have addressed in https://github.com/CS342/2023-PAWS/pull/4/commits/311574da1b1c62d47d6adea0ef0f3e96939d2c80.
To make the tests a bit more reliable, I have also increased the timeout here a bit: https://github.com/CS342/2023-PAWS/pull/4/commits/221b0bc6a904655184f8f84c7cfa1ce8e66627db
